pixart.js

```diff
@@ -1 +1,30 @@
 
+$(document).ready(function canvas() {
+    for (i = 0; i<= 5000; i++)
+{
+
+  $( "body" ).append($("<div class='square'></div>"))
+}
+      this.squares = $(".square")
+});
+
+var inputColor = {
+    this.color = this.input.val()
+    return this.color
+  }
+
+$("button").on("click", function setColor(){
+   this.inputColor.css("background", this.color)
```

Comment:
Does the jQuery button object come with a value called `inputColor`? That is what this code reads as.

Also, you want to replace the brush's background with whatever color the user enters into the input field. That color is not available in the button, meaning you will not be able to access it using `this` inside of this button click listener. You need to retrieve it from the input field.